factorio_noir/render.py

```diff
@@ -71,15 +69,40 @@ def matrix(self, saturation, brightness):
 DEFAULT_COLORSPACE = ColorSpace(X=0.3086, Y=0.6094, Z=0.0820)
 
 
-def apply_transforms(image, saturation, brightness):
+def apply_transforms(image, treatment):
     """Apply the needed transformations to the given image."""
     img_alpha = image.getchannel("A")
     img_rgb = image.convert("RGB")
 
-    color_space = DEFAULT_COLORSPACE.matrix(saturation, brightness)
+    color_space = DEFAULT_COLORSPACE.matrix(treatment.saturation, treatment.brightness)
 
     img_converted = img_rgb.convert("RGB", color_space)
+
+    for x_index, x_tile in enumerate(treatment.tiling):
```

Comment:
I know it's not much but `len(treatment.tiling)` is used a lot

Reply:
Ah, yes, I was originally going to move it out. (I moved y_max out, but didn't end up using it :| )